docs/quickstart.rst

```diff
@@ -107,15 +116,15 @@ we can evaluate metrics for subgroups within the data as below:
     >>> from fairlearn.metrics import MetricFrame
     >>> from sklearn.metrics import accuracy_score
     >>> from sklearn.tree import DecisionTreeClassifier
-    >>> 
+    >>>
     >>> classifier = DecisionTreeClassifier(min_samples_leaf=10, max_depth=4)
     >>> classifier.fit(X, y_true)
     DecisionTreeClassifier(...)
     >>> y_pred = classifier.predict(X)
-    >>> gm = MetricFrame(metrics=accuracy_score, y_true=y_true, y_pred=y_pred, sensitive_features=sex)
-    >>> print(gm.overall)
+    >>> as = MetricFrame(metrics=accuracy_score, y_true=y_true, y_pred=y_pred, sensitive_features=sex)
```

Comment:
I think `gm` is a remnant from what we used to call group metric(s) (?)

`as` is a word in itself... Maybe `mf` like elsewhere in the docs (?)

Reply:
> `as` is a word in itself.

I was pondering that problem too. 🤓 If the remainder of the docs use `mf`, I would suggest following suite in this case as well. `sr` as a special case for a second metric still makes sense that way.

Reply:
Ah, there's another one... Well, I don't think I'm going to advocate for `asmf` and `srmf` 😆 I guess it's fine as is, it's really a nitpicky request anyway... let me resolve this.